integr8ly/keycloak-operator
# dont send createonly to the API

the createOnly flag had been added to the API object, which was breaking the provision of createonly realms.

To verify this:
```
oc new-project sso-test
oc create -f deploy/rbac.yaml 
oc create -f deploy/operator.yaml 
oc create -f deploy/examples/keycloak_min.json 
oc create -f deploy/examples/keycloakRealm.json 
```
- See the realm is created correctly with users.
- Set a firstname for the users in the CR and save it
- See it is not reconciled into the user
- Delete the createOnly from the CR
- See the first name is reconciled.
